ako-operator/helm/ako-operator/crds/ako.vmware.com_hostrules.yaml

```diff
@@ -77,6 +77,20 @@ spec:
                     type: object
                   wafPolicy:
                     type: string
+                  tcpSettings:
+                    properties:
+                      listeners:
+                        items:
+                          properties:
+                            port:
+                              type: integer
+                            enableSSL:
+                              type: boolean
+                          type: object
+                        type: array
+                      loadBalancerIP:
```

Comment:
Should we include IP type also v4 /V6?

Reply:
Can user specify public ip over here for cloud deployments?

Reply:
How will it behave if EnablepublicIP is true and we have specified IP over here?

Reply:
We don't have ipv6 support yet, later we can possibly auto detect it from the field.
Not sure about the behaviour but programmatically, handling enablePublicIP and static IP is mutually exclusive.